tests.py

```diff
@@ -411,6 +433,16 @@ def test_obj_label_long_type_name(self):
             objgraph._obj_label(x, shortnames=False),
             r'mymodule\.MyClass\\n<mymodule\.MyClass object at .*')
 
+    def test_obj_attrs(self):
+        x = object()
+
+        self.assertRegex(
```

Comment:
I think `self.assertEqual` would work better here.  (Also, you can lose the blank line.)

Reply:
OK